lms/events/event.py

```diff
@@ -101,3 +102,33 @@ def _get_assignment_id(self):
             return assignment.id
 
         return None
+
+
+class AdminChangeEvent(BaseEvent):
```

Comment:
New event type.

It's on the generic side but making it only for `OrganizationEnabledChangedEvent` would be very similar code overall.